### PR TITLE
Updates for forceRefresh

### PR DESCRIPTION
**Pull request by shoatman**

### CODE REVIEW

File: common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java [hunk 1]
```diff
@@ -739,7 +744,7 @@ private AuthenticationConstants() {
          * Signature info for Azure authenticator app that installs authenticator
          * component.
          */
-        public static final String AZURE_AUTHENTICATOR_APP_SIGNATURE = "ho040S3ffZkmxqtQrSwpTVOn9r0=";
+        public static final String AZURE_AUTHENTICATOR_APP_SIGNATURE = "N1jdcbbnKDr0LaFZlqdhXgm2luE=";//
```
Comment (heidijinxujia) [inline common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java]: This is only for testing, right? Would it be better not to merge it?

Comment (iambmelt) [reply common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java]: +1

Comment (shoatman) [reply common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java]: I'll fix this.  We should probably revisit how this value gets set as part of the release process.  We effectively have the production signature checked in.

**Comment by shoatman** [comment]
@iambmelt @heidijinxujia - Thanks guys.  I've updated.

**Pull request merged**